mediation-old/src/main/java/com/wso2telco/dep/mediator/impl/ussd/USSDInboundHandler.java

```diff
@@ -162,6 +171,16 @@ public boolean handle(MessageContext context) throws CustomException, AxisFault,
 		return true;
 	}
 
+	
+	private void validateUssdAction(JSONObject jsonBody, MessageContext context) throws Exception {
```

Comment:
As mentioned in the sendussdhandler, validations needs to be moved to oneapi-validation.   
It seems this is wrong validation because ussd session will be continued through this handler. There can be all the ussd actions except "mtinit" ussd action.

Reply:
done